### PR TITLE
fix: broken json format for jwks

### PR DESCRIPTION
**Pull request by markusahlstrand**

### CODE REVIEW

File: src/authentication.ts [hunk 1]
```diff
@@ -131,9 +134,9 @@ async function isValidJwtSignature(
   const signature = new Uint8Array(
     Array.from(token.signature).map((c) => c.charCodeAt(0)),
   );
-  const jwkKeys = await getJwks(ctx.env, securitySchemeName);
+  const jwksKeys = await getJwks(ctx.env, securitySchemeName);
 
-  const jwkKey = jwkKeys.find((key) => key.kid === token.header.kid);
+  const jwkKey = jwksKeys.find((key) => key.kid === token.header.kid);
 
   if (!jwkKey) {
```
Comment (danielgent) [inline src/authentication.ts:141]: this variable should also be renamed right?
```suggestion
  if (!jwksKey) {
```

File: src/authentication.ts [hunk 2]
```diff
@@ -92,18 +93,20 @@ async function getJwks(env: Env, securitySchemeName: SecuritySchemeName) {
...
-      ? await env.TOKEN_SERVICE.fetch(jwksUrl)
-      : await fetch(jwksUrl);
+      if (!response.ok) {
+        throw new Error("Failed to fetch jwks");
+      }
 
-    if (!response.ok) {
-      throw new Error("Failed to fetch jwks");
-    }
+      const responseBody: { keys: JwkKey[] } = await response.json();
```
Comment (danielgent) [inline src/authentication.ts:106]: zod? If something is currently broke we could release this and then I could make any changes if you see fit :+1:

File: src/routes/tsoa/userinfo.ts [hunk 3]
```diff
@@ -1,8 +1,8 @@
 import { Controller, Get, Request, Route, Security, Tags } from "@tsoa/runtime";
 import { RequestWithContext } from "../../types/RequestWithContext";
 import { getDb } from "../../services/db";
-import { NotFoundError } from "src/errors";
-import { getId } from "src/models";
+import { NotFoundError } from "../../errors";
+import { getId } from "../../models";
```
[the remark this answers is not in the document]
Comment (markusahlstrand) [reply src/routes/tsoa/userinfo.ts:5]: It actually worked but I get squiggles locally

File: src/authentication.ts (hunk 2 above)
Comment (markusahlstrand) [reply src/authentication.ts:106]: Yes.. this repo does actually have zod already so would be great :)

**Pull request merged**